docs/index.asciidoc

```diff
@@ -41,3 +41,44 @@ Notes:
 * the fluent uses a second-precision time for events, so you will never see
   subsecond precision on events processed by this codec.
```

Comment:
I know that this was pre-existing, but we can improve it by using this instead: 

> NOTE: The fluent uses a second-precision time for events, so you will never see subsecond precision on events processed by this codec.

Reply:
Also, it seems like a word is missing.  "NOTE: The fluent ________ uses a second-precision time..." Should that be the fluent schema? The fluent codec?

Reply:
if it's missing something than it would be the msgpack schema but it's a fluent specific use here ... would this work?
```suggestion
NOTE: Fluent uses second-precision for events, so you will not see sub-second precision on events processed by this codec.
```
(honestly not even sure if that statement is true anymore)

Reply:
Your call. We should move forward to get your other important changes in. Perhaps we open an issue for tracking the validity of the statement? 

I've approved, and I trust your judgement on how to proceed. Lemme know if you'd like for me to open an issue.